cfme/infrastructure/host.py

```diff
@@ -634,7 +635,14 @@ def create_from_config(self, host_key):
 
 @navigator.register(HostsCollection)
 class All(CFMENavigateStep):
-    VIEW = HostsView
+
+    @property
+    def VIEW(self):
```

Comment:
please use  `# noqa` its not as per definition standard.

Reply:
will do. But, what is that? sorry I don't know.